OptimizelySDKCore/OptimizelySDKCore/OPTLYBaseCondition.m

```diff
@@ -87,6 +87,11 @@ - (nullable NSNumber *)evaluateMatchTypeExact:(NSDictionary<NSString *, NSObject
     else if ([self.value isValidBooleanAttributeValue] && [userAttribute isValidBooleanAttributeValue]) {
         return [NSNumber numberWithBool:[self.value isEqual:userAttribute]];
     }
+    
+    // Log Invalid Attribute Value Type
+    NSString *userAttributeClassName = NSStringFromClass([userAttribute class]) ?: @"null";
```

Comment:
`?: @"null"` definitely saves some lines of code, but in the case of a null value it'd be better to change the entire message to the smoother-reading `OPTLYLoggerMessagesAudienceEvaluatorConditionEvaluatedAsUnknownForUnexpectedTypeNull`. And more importantly, that log message should be at debug level instead of warning level.

Same thing earlier in this function, although you might be able to get rid of that particular code block since we can just fall through to the code you've added here.